designer/client/src/components/graph/node-modal/editors/expression/DictParameterEditor/DictParameterEditor.tsx

```diff
@@ -74,7 +74,7 @@ export const DictParameterEditor: ExtendedEditor<Props> = ({
     const isValid = isEmpty(fieldErrors);
 
     return (
-        <Box className={nodeValue}>
+        (<Box className={nodeValue}>
```

Comment:
Do we need it? It looks like your IDE autoformatting settings, because prettier just remove this parenthesis

Reply:
I think this was generated by mui's  codemods. It seems to work fine without parentheses, so I removed them here and other places where they had been added.